src/main/java/org/osc/sdk/controller/FlowInfo.java

```diff
@@ -16,15 +16,49 @@
  *******************************************************************************/
 package org.osc.sdk.controller;
 
+/**
+ * This class represents a flow information.
```

Comment:
Provide more details here... i.e.: represents a flow information to be used when....